### DISCUSSION TITLE
collapse some menu items for better

### DISCUSSION THREAD

**Discussion by emics** [magicbug/Cloudlog#3148] [Ideas / Feature Requests]
using Cloudlog from an old laptop, that has a resolution of 1024x768, the menu is cut off and you have to scroll down the page to see all the items and you often lose the focus of the menu.

![menu_cutted](https://github.com/magicbug/Cloudlog/assets/1004363/bcd6ccbb-abc4-486e-9c6c-d15321d91e98)

Could it be useful to collapse all the items relating to _External Logbooks_ into a single entry (like _Other Export Options_ item)?
If necessary I can take care of it directly.

**Comment by magicbug** [comment]
I see no reason why third party couldn't be a "Third Party Logbooks" sub menu would probably make good sense

**Comment by emics** [comment]
I think some icon is wrong,
Here a little proposal, any suggest is apreciated :)

logbook
![logbook](https://github.com/magicbug/Cloudlog/assets/1004363/b8f7ad2b-13fe-4df3-983b-b9675f13efe6)

qso
![qso](https://github.com/magicbug/Cloudlog/assets/1004363/a6f5b508-68ab-4659-8c9f-80768157a9bd)

analytics
![analytics](https://github.com/magicbug/Cloudlog/assets/1004363/34294a38-b10c-4396-8076-e81881d0bfd8)

admin
![admin](https://github.com/magicbug/Cloudlog/assets/1004363/a12f7fb8-b517-4eaf-9dbf-d325bfa59ade)

user
![user](https://github.com/magicbug/Cloudlog/assets/1004363/7d73fd8a-165f-4af7-8ddd-015f859bd515)
